packages/mobile/src/pincode/authentication.ts

```diff
@@ -46,6 +46,7 @@ export const PIN_LENGTH = 6
 // Pepper and pin not currently generalized to be per account
 // Using this value in the caches
 export const DEFAULT_CACHE_ACCOUNT = 'default'
+export const CANCELED_PIN_INPUT = 'CANCELED_PIN_INPUT'
```

Comment:
nit: Looks like `cancelled` is British English and `canceled` is US English, though we're using `cancelled` everywhere.
```suggestion
export const CANCELLED_PIN_INPUT = 'CANCELLED_PIN_INPUT'
```